docs/digital/exports/main-export-fields.md

```diff
@@ -207,6 +206,8 @@ Other custom data can be present in the thread export depending on the source ty
 | content.destroyed                                       | A message was deleted                                                      |
 | content.discussion_initiated                            | A discussion was initiated by an agent                                     |
 | content.discussion_planned                              | A discussion was planned by an agent                                       |
+| content.draft_created                                   | A draft was created by an agent                                            |
+| content.draft_updated                                   | A draft was updated by an agent                                            |

```

Comment:
you should add the discarded event as well so that you don't have to do another PR just for it

Reply:
done in [d73a84b](https://github.com/ringcentral/engage-digital-api-docs/pull/263/commits/d73a84b48b4090ec3dc7ee90f694545141c3222c)